docs/scripts/installpostgres.sh

```diff
@@ -10,5 +10,10 @@
 
 set -e # to exit if a command in the script fails
 
-# install PostgreSQL 16
-sudo apt install postgresql -y
+# install PostgreSQL 15
+sudo apt install wget ca-certificates -y
```

Comment:
As we have curl already, please replace wget with curl to avoid another unnecessary dependency.

Reply:
Done